Appl/IDialup/IDInfo.goc

```diff
@@ -471,7 +471,13 @@ void ABLog(char *fmt, ...)
 	    @send IDIdleNotifyDialog::MSG_GEN_SET_NOT_USABLE(VUM_NOW);
 	    @send IDLIANotifyDialog::MSG_GEN_INTERACTION_ACTIVATE_COMMAND(IC_DISMISS);
 
-	    AccessPointSetActivePoint(defaultAccpnt, APT_INTERNET);
+			/* why are we calling this here?!
+			   This probably breaks things with G_activeAccpnt and
+				 currentAccpnt - can't test however, no dialup available.
+				 But it really shouldn't be needed, so let's be brave
+				 and disable it... Falk, don't kill me.
+				 (meyerk 01/2025) */
+	    // AccessPointSetActivePoint(defaultAccpnt, APT_INTERNET);

```

Comment:
As discussed, just leave this here as it was, if this is not actually causing problems.